study-builder/studies/atcp/medical-history.conf

```diff
@@ -0,0 +1,6111 @@
+                    "isRestricted": false,
+                    "isDeprecated": false,
+                    "hideNumber": true,
+                    "placeholderTemplate": null,
+                    "additionalInfoHeaderTemplate": null,
+                    "additionalInfoFooterTemplate": null,
+                    "validations": [],
+                    "suggestionType": "NONE",
+                    "suggestions": [],
+                    "promptTemplate": {
+                      "templateType": "HTML",
+                      "templateText": "$city",
+                      "variables": [
+                        {
+                          "name": "city",
+                          "translations": [
+                            {
+                              "language": "en",
+                              "text": ${i18n.en.medical_history_where_made_diagnosis_city}
+                            }
+                          ]
+                        }
+                      ]
+                    }
+                  },
+                  {
+                    "stableId": "STATE",
+                    "questionType": "TEXT",
+                    "inputType": "TEXT",
+                    "isRestricted": false,
+                    "isDeprecated": false,
+                    "hideNumber": true,
+                    "placeholderTemplate": null,
+                    "additionalInfoHeaderTemplate": null,
+                    "additionalInfoFooterTemplate": null,
+                    "validations": [],
+                    "suggestionType": "NONE",
+                    "suggestions": [],
+                    "promptTemplate": {
+                      "templateType": "HTML",
+                      "templateText": "$state",
+                      "variables": [
+                        {
+                          "name": "state",
+                          "translations": [
+                            {
+                              "language": "en",
+                              "text": ${i18n.en.medical_history_where_made_diagnosis_state}
+                            }
+                          ]
+                        }
+                      ]
+                    }
+                  },
+                  {
+                    "stableId": "COUNTRY",
+                    "questionType": "TEXT",
+                    "inputType": "TEXT",
+                    "isRestricted": false,
+                    "isDeprecated": false,
+                    "hideNumber": true,
+                    "placeholderTemplate": null,
+                    "additionalInfoHeaderTemplate": null,
+                    "additionalInfoFooterTemplate": null,
+                    "validations": [],
+                    "suggestionType": "NONE",
+                    "suggestions": [],
+                    "promptTemplate": {
+                      "templateType": "HTML",
+                      "templateText": "$country",
+                      "variables": [
```

Comment:
We should probably have an autocomplete or autosuggest for country, but we don't at this time. We have implemented them before thought:
https://github.com/broadinstitute/ddp-study-server/blob/12ce673b3d6a4eafaa312af3d84e3004aa87517a/study-builder/studies/angio/about-you.conf#L2706